tests/plugins/tasks/commands/test_command.py

```diff
@@ -2,13 +2,18 @@
 from brigade.plugins.tasks import commands
 
 
+def echo_hostname(task):
+    command = "echo {host.name}".format(host=task.host)
+    task.run(task=commands.command, command=command)
+
+
 class Test(object):
 
     def test_command(self, brigade):
-        result = brigade.run(commands.command, command="echo {host.name}")
+        result = brigade.run(echo_hostname)
         assert result
         for h, r in result.items():
-            assert h == r.stdout.strip()
+            assert h == r[1].stdout.strip()
```

Comment:
This seemed a bit strange...why the addition of the [1] i.e. why did it go from `r.stdout.strip()` to `r[1].stdout.strip().

Reply:
Okay, I understand this behavior now.

Reply:
Just for reference; the reason is because instead of calling a task directly we are calling now a grouped task. Grouped tasks return a dict of results (key is host) with a list of results of which the first element is the result for the whole group.